test/CMakeLists.txt

```diff
@@ -101,6 +111,17 @@ if (pybind11_VERSION VERSION_LESS 2.5.99)
  list(REMOVE_ITEM binder_tests T11.override)
  list(REMOVE_ITEM binder_tests T15.inner_class)
 endif()
+if( CMAKE_HOST_SYSTEM_NAME MATCHES "Darwin" )
+ list(REMOVE_ITEM binder_tests T42.stl.names.multimap)
+ list(REMOVE_ITEM binder_tests T42.stl.names.multiset)
+ list(REMOVE_ITEM binder_tests T42.stl.names.set)
+ list(REMOVE_ITEM binder_tests T42.stl.names.map)
+ list(REMOVE_ITEM binder_tests T42.stl.names)
+ list(REMOVE_ITEM binder_tests T40.stl)
+ list(REMOVE_ITEM binder_tests T11.override)
+ list(REMOVE_ITEM binder_tests T10.inheritance)
+ list(REMOVE_ITEM binder_tests T02.function)
+endif()
```

Comment:
out of curiosity: - what exactly wrong with these tests on Mac? Is it something that can be fixed?

Reply:
There is a clash of different system headers. Something like pre C++11 vs post C++11. One should set the paths correctly and some flags to make is disapear.  But this problem  is relevant only for older macs, e.g. my working 10.13, things should work fine on 10.15.  I'm going to request a newer machine, but that is my tertiary system, so, well, I survive as it is.
 In short, that is not binder problem.